### DISCUSSION TITLE
Team Call #2

### DISCUSSION THREAD

**Discussion by kaspth** [kaspth/oaken#33] [Team Calls]
Hey everybody, let's do another call!

This'll be on Thursday September 7th at 7pm UTC+2.

-------

Kasper Timm Hansen is inviting you to a scheduled Zoom meeting.

Topic: Kasper Timm Hansen's Personal Meeting Room

Join Zoom Meeting
https://us06web.zoom.us/j/7661417948?pwd=8eJZyXfbqPfnEMjWK6YD62tzBbLhA8.1

Meeting ID: 766 141 7948
Passcode: xPY1T7

---

One tap mobile
+12532050468,,7661417948#,,,,*338646# US
+12532158782,,7661417948#,,,,*338646# US (Tacoma)

---

Dial by your location
• +1 253 205 0468 US
• +1 253 215 8782 US (Tacoma)
• +1 301 715 8592 US (Washington DC)
• +1 305 224 1968 US
• +1 309 205 3325 US
• +1 312 626 6799 US (Chicago)
• +1 346 248 7799 US (Houston)
• +1 360 209 5623 US
• +1 386 347 5053 US
• +1 507 473 4847 US
• +1 564 217 2000 US
• +1 646 558 8656 US (New York)
• +1 646 931 3860 US
• +1 669 444 9171 US
• +1 689 278 1000 US
• +1 719 359 4580 US
• +1 720 707 2699 US (Denver)

Meeting ID: 766 141 7948
Passcode: 338646

Find your local number: https://us06web.zoom.us/u/kdNhg6BLG

**Comment by kaspth** [comment]
See you in 4.5 hours! 😄

**Reply by larkro** [reply]
Would love to, but some family logistics rearranged the evening. :-( Don't think I can't make it. Will try!

**Reply by tcannonfodder** [reply]
I also can't because of work; but definitely want to continue the conversation on #32

**Comment by manuelfcreis** [comment]
Oh damn I just realised I'm one hour late 🤦🏻  I misread the time

**Comment by salochara** [comment]
Oh 🙃 I also misread the time!
